### DISCUSSION TITLE
请问ios版本在哪里下载

### DISCUSSION THREAD

**Discussion by desperadoxhy** [isaced/V2exOS#28] [Q&A]
没看到ios地址

**Comment by isaced** [comment]
iOS 目前处于开发阶段，~~还没有正式发布~~ 目前 TestFlight 分发

**Reply by kid1412621** [reply]
老板 TestFlight 过期了，弃坑了吗😂

**Reply by isaced** [reply]
> 老板 TestFlight 过期了，弃坑了吗😂

已更新～

下次如果过期可以提醒我一下，有时候会忘掉

**Comment by isaced** [comment]
1. 在 [Releases](https://github.com/isaced/V2exOS/releases/tag/iOS_1.0) 中可以下载 `.iap` 进行自签名安装。
2. [TestFlight](https://testflight.apple.com/join/MOtKdxHH) 安装。（过期了需要等我重新构建一下）
